lib/edge_impulse/ei_ncs.cpp

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <assert.h>
+#include <ei_run_classifier.h>
+#include <ei_ncs.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ei_ncs, CONFIG_EI_NCS_LOG_LEVEL);
+
+#define INPUT_FRAME_SIZE	EI_CLASSIFIER_RAW_SAMPLES_PER_FRAME
+#define INPUT_WINDOW_SIZE	EI_CLASSIFIER_DSP_INPUT_FRAME_SIZE
+
+#define PROCESS_BUF		1
```

Comment:
I don't get it? Won't it be easier to use `bool` instead?

Reply:
I will add dedicated value that will be used to trigger processing, that one will no longer be needed